zip-seanbowe-esktransfer.rst

```diff
@@ -0,0 +1,70 @@
+::
+
+  ZIP: 212
+  Title: Transfer Sapling Ephemeral Secret to Recipient in Note Plaintext
```

Comment:
```suggestion
  Title: Allow recipient to derive Sapling ephemeral secret from note plaintext
```
(since we're not proposing to literally include the secret in the note plaintext any more).